src/gui/preferences.c

```diff
@@ -1454,7 +1454,7 @@ static void edit_preset(GtkTreeView *tree, const gint rowid, const gchar *name,
   gtk_widget_set_hexpand(GTK_WIDGET(g->focal_length_max), TRUE);
 
   // raw/hdr/ldr
-  label = gtk_label_new(_("format"));
+  label = gtk_label_new(C_("raw/hdr/ldr", "format"));
```

Comment:
C_("file format", "format") ?

Reply:
Neither RAW, LDR nor HDR is really a file format, as opposed to e.g. JPG or PDF. This is one example where English easily gets by using a rather broad term ("format"), but other languages might need to specify whether it is a "file format" (like JPG) or something to in the vein of "image type" (like HDR).

Reply:
Other languages, which one? Do you need it? I really want to have a minimal change at this stage. The minimum for you to get a clean translation and as I said with the more generic context as possible otherwise we'll just end up with lot of more translated string to do and this in all languages.

Reply:
The specific "other" language I had in mind here was of course Estonian, where I would very much like to be able to translate this occurrence differently from the generic "format".
I checked Wikipedia articles about RAW and HDR in e.g. Danish, Norwegian, Swedish, Finnish and German - RAW was said to be either a "common denominator" for various file formats or just file format (so based on that RAW actually could be argued to fall under the generic "format" as well), however HDR was mostly called a "technique". So translators of at least those languages might also like the opportunity to translate this hdr/ldr/raw "format" differently from the generic format.